R CMD BUILD error

[R-hub 🌐 / ubuntu-gcc12 (r-devel-linux-x86_64-debian-gcc)](https://github.com/insightsengineering/tern.mmrm/actions/runs/11537370491/job/32325099262#logs)

[R-hub 🌐 / donttest](https://github.com/insightsengineering/tern.mmrm/actions/runs/11537370491/job/32325102178#logs)

```
── R CMD build ─────────────────────────────────────────────────────────────────
* checking for file ‘.../DESCRIPTION’ ... OK
* preparing ‘tern.mmrm’:
* checking DESCRIPTION meta-information ... OK
* installing the package to build vignettes
* creating vignettes ... ERROR
--- re-building ‘introduction.Rmd’ using rmarkdown
Quitting from lines 100-112 [unnamed-chunk-4] (introduction.Rmd)
Error: processing vignette 'introduction.Rmd' failed with diagnostics:
.onLoad failed in loadNamespace() for 'TMB', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/github/home/R/x86_64-pc-linux-gnu-library/4.5/TMB/libs/TMB.so':
  libomp.so.5: cannot open shared object file: No such file or directory
--- failed re-building ‘introduction.Rmd’
SUMMARY: processing the following file failed:
  ‘introduction.Rmd’
Error: Vignette re-building failed.
Execution halted
Error in proc$get_built_file() : Build process failed
Calls: <Anonymous> ... build_package -> with_envvar -> force -> <Anonymous>
Execution halted
Error: Process completed with exit code 1.
```

--- a/DESCRIPTION
+++ b/DESCRIPTION
@@ -53,6 +53,7 @@ Suggests:
     Matrix,
     rmarkdown (>= 2.23),
     testthat (>= 3.1),
+    TMB,
     vdiffr (>= 1.0.7),
     withr (>= 2.0.0)
 VignetteBuilder:

